### PR TITLE
Add missing optional packages to `requirements/*.txt`

### PR DESCRIPTION
**Pull request by akihironitta**
## What does this PR do?
As reported in https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/444#issuecomment-743010814, the current CI lacks `matplotlib` in the environment.

## Before submitting
- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together? Otherwise, we ask you to create a separate PR for every change.
- [x] Did you make sure to update the documentation with your changes?
- [x] Did you write any new necessary tests?
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/CHANGELOG.md)?

## PR review
 - [x] Is this pull request ready for review? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: requirements/loggers.txt [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # test_tube>=0.7.5
-# trains>=0.14.1
\ No newline at end of file
+# trains>=0.14.1
+matplotlib
+wandb
```
Comment (Borda) [inline requirements/loggers.txt:4]: where is this used?

Comment (akihironitta) [reply requirements/loggers.txt:4]: `grep wandb -rn pl_bolts`
```
pl_bolts/callbacks/data_monitor.py:14:    import wandb
pl_bolts/callbacks/data_monitor.py:16:    wandb = None
pl_bolts/callbacks/data_monitor.py:91:                data={name: wandb.Histogram(tensor)}, commit=False,
```

Comment (Borda) [reply requirements/loggers.txt:4]: ok, is wandb the only one supporting histograms? I think that it shall be logger agnostic...
anyway, lest move it among logger? 
https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/requirements/loggers.txt

Comment (akihironitta) [reply requirements/loggers.txt:4]: so, shall we add `wandb` to requirements for now?

**Review by Borda — APPROVED** [review]

**Pull request merged**